tests/tests.rs

```diff
@@ -194,6 +196,14 @@ fn compare_generated_header(
         ),
     };
 
+    let flags = if check_roundtrip {
```

Comment:
It looks like `flags` is not used if we are not doing the roundtrip test.
Could you move the definition under `if check_roundtrip { ...`?

Reply:
I can't, because `builder` is moved below in the `generate()` call.